sql/analyzer/aliases.go

```diff
@@ -212,3 +227,87 @@ func normalizeExpression(tableAliases TableAliases, e sql.Expression) sql.Expres
 
 	return normalized
 }
+
+// renameAliasesInExpressions returns expressions where any table references are renamed to the new table name.
+func renameAliasesInExpressions(expressions []sql.Expression, oldNameLower string, newName string) ([]sql.Expression, error) {
+	for i, e := range expressions {
+		newExpression, tree, err := renameAliasesInExp(e, oldNameLower, newName)
```

Comment:
most of the rest of the code uses this sort of pattern:
```golang
newExpression, same, err := renameAliasesInExp(e, oldNameLower, newName)		if err != nil {
			return nil, err
		}
		if !same {
			expressions[i] = newExpression
		}
		```

Reply:
Updating to use the `same` pattern.